.drone.star

```diff
@@ -129,7 +129,9 @@ def main(ctx):
     notify(ctx),
   ]
 
-  if '[docs-only]' in (ctx.build.title + ctx.build.message):
+  if ctx.build.cron != "":
+    pipelines = before
```

Comment:
We could also do the `notify_pipelines` logic, like for the docs-only builds.
Then the drone crn will notify to rocket chat if something fails.